tests/waku_rln_relay/test_waku_rln_relay.nim

```diff
@@ -710,61 +710,6 @@ suite "Waku rln relay":
       msgValidate3 == MessageValidationResult.Valid
       msgValidate4 == MessageValidationResult.Invalid
 
-  asyncTest "should validate invalid proofs if bandwidth is available":
```

Comment:
Ooh what a pitty. Why we are removing that? I am happy with it although not fully understanding the test logic ;P

Reply:
this pr also removes the bandwidth constraint validator, which should be a separate validator!